packages/babel-compat-data/scripts/build-modules-support.js

```diff
@@ -35,8 +35,8 @@ function process(source) {
   const allowedBrowsers = {};
 
   Object.keys(stats).forEach(browser => {
-    const browserName = browserNameMap[browser] ?? browser;
-    let browserSupport = stats[browserSupportMap[browserName] ?? browser];
+    const browserName = browserNameMap[browser] || browser;
```

Comment:
Adjusted this since we don't compile this script and `??` isn't supported until v14?

Reply:
😛 I went ahead and assumed core members are on node.js 14 now.

Reply:
I remember other projects just add some node code somewhere to throw + error message if not on certain node version? we could do that too